include/inviwo/qt/editor/welcomewidget.h

```diff
@@ -67,14 +68,18 @@ class IVW_QTEDITOR_API WelcomeWidget : public QSplitter {
     void updateRecentWorkspaces(const QStringList& list);
     void enableRestoreButton(bool hasRestoreWorkspace);
 
+    void updateLoadIcon(bool append, bool openWithPath);
```

Comment:
This could be made clearer as it is in the public interface. Perhaps leave the logic to the caller and instead allow the icon itself to be specified?
Also, perhaps `setLoadWorkspaceIcon`?